doc/api/external_formats.md

```diff
@@ -0,0 +1,78 @@
+# Modules: External formats
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental
+
+## Introduction
+
+Node natively understands a handful of formats (see the table in [Modules: Load hook][Load
+customization hook]). Non-native / external formats require a translator (under the hood, this is a
+custom loader). When attempting to run a module with an external format (ex `node main.ts`), node
+will try to detect what it is. At current, when node is able to identify the format, it prints a
+message to stdout with instructions (that lead here).
+
+## Setting up a module translator
+
+There are a few, currently manual, steps you need to complete. We intend to automate these in
+future.
+
+1. You will need to find and install a module translator via your preferred package manager. Some
+   package managers provide a CLI utility as well as a website.
+1. Once installed, in order to get node to automatically use it, you will need to create a `.env`
+   file containing a [`--import`][CLI import] for your chosen module translator's package (ex `NODE_OPTIONS="--import=example-translator/register"`). See [CLI:
+   env file][CLI env file].
+1. When you subsequently run node, include the env file flag (ex `node --env-file=.env main.ts`).
```

Comment:
Because this instructs the user to manually complete a process in the same way it will be automated in future (so no migration will be needed when node starts automatically loading `./.env`). Doing it this way, eventually the user can stop doing something instead of change what they need to do.